remoteappmanager/static/js/admin/vue-components/ContainersView.js

```diff
@@ -1,96 +1,99 @@
+        '    :globalActions="table.globalActions"' +
+        '    :rowActions="table.rowActions">' +
+        '  </data-table>' +
+        '  <confirm-dialog ' +
+        '    v-if="stopContainerDialog.show"' +
+        '    title="Stop container"' +
+        '    :okCallback="stopContainer"' +
+        '    :closeCallback="closeStopContainerDialog">' +
+        '    <div>Do you want to stop container {{ stopContainerDialog.containerToStop }}?</div>' +
+        '  </confirm-dialog>' +
+        '</adminlte-box>',
       data: function () {
         return {
-          containers: [],
-          showStopContainerDialog: false,
-          communicationError: null,
-          containerToStop: null,
+          table: {
+            headers: ["URL ID", "User", "Image", "Docker ID", "Mapping ID"],
+            rows: [],
+            rowActions: [
+              {
+                label: "Stop",
+                callback: this.stopAction
+              }
+            ]
+          },
+          stopContainerDialog: {
+            show: false,
+            containerToStop: null
+          },
+          communicationError: null
         };
       },
       mounted: function () {
         this.updateTable();
       },
       methods: {
         updateTable: function() {
+          var self = this;
           this.communicationError = null;
           resources.Container.items()
             .done(
-              (function (identifiers, items) {
-                var containers = [];
+              function (identifiers, items) {
+                self.table.rows = [];
                 identifiers.forEach(function(id) {
                   var item = items[id];
-                  item.identifier = id;
-                  containers.push(item);
+                  self.table.rows.push([
+                    id,
+                    item.user,
+                    item.image_name,
+                    item.docker_id,
+                    item.mapping_id]);
                 });
-                this.containers = containers;
-              }).bind(this))
+              })
             .fail(
-              (function () {
-                this.communicationError = "The request could not be executed successfully";
-              }).bind(this)
+              function () {
+                self.communicationError = "The request could not be executed successfully";
+              }
             );
         },
-        containerStopped: function() {
-          this.showStopContainerDialog = false;
-          this.updateTable();
+        stopAction: function(row) {
+          this.stopContainerDialog.containerToStop = row[0];
+          this.stopContainerDialog.show = true;
         },
-        stopAction: function(index) {
-          this.containerToStop = this.containers[index].identifier;
-          this.showStopContainerDialog = true;
+        stopContainer: function () {
+          if (this.stopContainerDialog.containerToStop === null) {
```

Comment:
not in the current code, but if someone decides to call it programmatically and does not fill containerToStop it will just do nothing, although probably it's better if we let the error happen.